commit 355e3e4ccc2cd4b29cc37546474448a8f02e3bbc
Author: Candice Li <candice.li@amd.com>
Date:   Mon Aug 23 15:17:35 2021 +0800

    drm/amd/amdgpu: add name field back to ras_common_if
    
    Adding name field back to ras_common_if to work around error
    injection failure with amdgpuras tool.
    
    Signed-off-by: Candice Li <candice.li@amd.com>
    Reviewed-by: John Clements <john.clements@amd.com>
    Signed-off-by: Alex Deucher <alexander.deucher@amd.com>

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_ras.h b/drivers/gpu/drm/amd/amdgpu/amdgpu_ras.h
index abc5710898e8..5b5163357fcb 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_ras.h
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_ras.h
@@ -309,6 +309,7 @@ struct ras_common_if {
 	enum amdgpu_ras_block block;
 	enum amdgpu_ras_error_type type;
 	uint32_t sub_block_index;
+	char name[32];
 };
 
 struct amdgpu_ras {